### PR TITLE
Throw errors instead of strings

### PR DESCRIPTION
**Pull request by william42**
When you're running tests and they fail, it's nice to get stack traces instead of a message telling you to throw errors instead of strings.

### CODE REVIEW

File: src/variety/refactor.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (x-sheep) [inline src/variety/refactor.sh:1]: I don't think any of these shell files need to be here.

File: src/puzzle/Bank.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ pzpr.classmgr.makeCommon({
 		setData: function(value, index) {
 			var len = this.board.bank.pieces.length;
 			if (index < 0 || index > len) {
-				throw "Index out of range";
+				throw Error("Index out of range");
```
Comment (x-sheep) [inline src/puzzle/Bank.js:216]: Shouldn't this be `throw new Error` instead?

**Review by x-sheep — CHANGES_REQUESTED** [review]

File: src/variety/refactor.sh (hunk 1 above)
Comment (william42) [reply src/variety/refactor.sh:1]: oops, didn't mean to commit those

File: src/puzzle/Bank.js (hunk 2 above)
Comment (william42) [reply src/puzzle/Bank.js:216]: `throw Error` and `throw new Error` are identical in behavior per the ES spec: https://262.ecma-international.org/7.0/#sec-error-constructor

You could argue the latter is better on principle but the code seemed to use the former in a few places already.

**Pull request merged**